### DISCUSSION TITLE
The CMB, axis of evil, and the Oort Cloud

### DISCUSSION THREAD

**Discussion by mikehelland** [a-cosmology-group/acg#217] [Astrophysics and Physics]
_Background:_ Alternative cosmologies need an explanation of the CMB. The mainstream explanation has a problem known as the [axis of evil](https://en.wikipedia.org/wiki/Axis_of_evil_(cosmology)) which aligns the CMB with the solar system.

----

The suggestion here is that the CMB is not a cosmological phenomenon, but actually has its origin in the solar system. Here's a couple pages I found. They don't seem to be written by credible authors, but I'm curious if anyone has thoughts on these.

[https://ui.adsabs.harvard.edu/abs/20...1006P/abstract](https://ui.adsabs.harvard.edu/abs/2021APS..APRZ21006P/abstract)
**Cosmic Microwave Background Radiation is NOT a Relic of the Big Bang**
Prevenslik, Thomas
APS April Meeting 2021, abstract id.Z21.006

>The CMB evolved from the Penzias and Wilson discovery of microwave radiation coming from space. Today, the CMB temperature at 2 . 725 +/- 0 . 00057 K is thought to be a relic of the Big Bang. However, the CMB may only be microwave emission from debris in the Oort Cloud. Unlike the disk-like Planetary System in the ecliptic of the Sun, the Oort cloud is spherical allowing the emission of EM radiation observed by Penzias and Wilson to be uniform in all directions. At 10,000 AU, the Oort cloud at 2.725 K varies by +/- 0 . 00057 K spread over about 12 AU. Because the Planck law denies atoms at a given temperature under EM confinement the heat capacity to conserve heat by a change in temperature, a debris temperature of 2.725 K is only possible for solar radiation having wavelengths λ > 20,000 microns where atoms do indeed have classical and finite heat capacity. Otherwise, solar heat is conserved by creating standing EM waves across the debris diameters d, where d > λ/2n. For refractive index n = 1.5, the minimum d > 7 m which means more than 8 billion asteroids in the Oort cloud emit discrete CMB radiation that appears spotted on WMAP and Planck surveys. Since the CMB of the Big Bang would be uniform and not spotted, the CMB is NOT a relic of the Big Bang

And here's another guys take on it:

http://aoi.com.au/OortSoup/index.htm
**P2: The Oort Soup as the real origin of Cosmic Microwave Background Radiation**

Something else to consider would be the heliopause. We learned, somewhat recently, that it's much hotter there than thought:

[https://ui.adsabs.harvard.edu/abs/20...1019R/abstract](https://ui.adsabs.harvard.edu/abs/2019NatAs...3.1019R/abstract)

>Voyager 2 observations show that the temperature is 30,000-50,000 K, whereas models and observations predicted a VLISM temperature of 15,000-30,000 K. 

Here's someone trying to connect the two:

https://arxiv.org/abs/0905.2978
**A Heliosheath Model for the Origin of the CMB Quadrupole Moment**

Any thoughts on these ideas about a local origin for the CMB? 

(PS, If your theory of the CMB is not related to the solar system, this is not the thread for them.)

**Comment by RedshiftDrift** [comment]
T. Prevenslik's redshift model is listed on [the ACG website](https://cosmology.info/redshift/#light---dust-interaction)  However hard somenoe may try, dust does not have the correct properties to imitate the cosmological redshift.  Moreover, so much dust is needed, distant galaxies would not be visible.  The paper says "Because the Planck law denies atoms at a given temperature under EM confinement the heat capacity to conserve heat by a change in temperature..."  This is nonsense, Planck's law doesn't 'deny' anything.

--------------

I've never heard of David Noel and his Oort soup.  Looking at his website, "The Oort Soup -- most of the Universe" is another 'CMB generated by dust' hypothesis.  It doesn't work for the same reasons given above.

--------------

I didn't know about that work, H.N. Sharpe is on the right track with his "Heliosheath Model for the Origin of the CMB Quadrupole Moment"  This was discuted at length on the ACosmologyGroup@gaggle.mail forum [private] in early 2023, just before continuing the discussions on github. (Were you a member then @mikehelland ?)

**Reply by mikehelland** [reply]
Thanks @RedshiftDrift .

I'm not interested in Prevenslik's redshift model (I have my own), just the possibility of an Oort cloud - CMB connection. The standing waves bit in particular. I'm not familiar with how that would work (or if it even does).

I wasn't part of the discussion prior to finding this github repo. I'm very grateful for it!

**Comment by mikehelland** [comment]
I'm still pretty interested in this.

Can we take from Voyager 2 that the solar system is surrounded by a ~40,000 K plasma?

>The VLISM is variable near the HP and hotter than expected. Voyager 2 observations show that the temperature is 30,000-50,000 K, whereas models and observations predicted a VLISM temperature of 15,000-30,000 K.

https://ui.adsabs.harvard.edu/abs/2019NatAs...3.1019R/abstract

This is an empirical measurement... right?

Can this be detected from Earth? 

This seems to make a good candidate for the origin of the CMB. It's aligned with the solar system (axis of evil). 

https://en.wikipedia.org/wiki/Axis_of_evil_(cosmology)

The CMB has a cold spot, to the south. Would that be the "tail" of the heliopause? This shows where the Voyagers went.

![image](https://github.com/a-cosmology-group/acg/assets/19311107/f7786f3e-4b28-4fff-9f4b-c0e1a6dd1d7c)

I presume that's supposed to be the sun in the middle, showing the orbit of the planets around it. But shouldn't the orbits be up and down in this image? I thought the Earth travels around the sun more like this:

![image](https://github.com/a-cosmology-group/acg/assets/19311107/f7f314b7-4f11-4866-9ee0-01a0ce502ea5)

So both of the Voyagers went "north". But the cold spot is "south."

To test if the CMB is actually coming from the heliopause/sphere/sheath, we could just send a probe in that direction. If it ends up being slightly cooler there... CMB cold spot explained?

Is there something obvious I'm missing? This seems like a pretty good, non-cosmological source of the CMB.

**Reply by HanDeBruijn** [reply]
Picture taken from a website on [HELIOPHYSICS](http://www.tp4.rub.de/heliophysics/en/research.php):

![web_sketch-heliosphere](https://github.com/a-cosmology-group/acg/assets/135433222/a8331de2-ed20-48df-a759-61a3d394fc65)

**Note.** Horst Fichtner is a friend of mine. I have [collaborated](http://www.alternatievewiskunde.nl/zonwind/index.htm) with him in the past (1996).

**Reply by mikehelland** [reply]
I don't get why the planet's orbits would be depicted that way, in your image, when they should be at as ~80 degree angle to the "heliotail", which is what I would expect from the "Side View" image in my post..

**Reply by Francois-Zinserling** [reply]
I now understand something I did not an hour ago. Both images are correct. 
Heliosphere does not depict a headwind. Bow-wave and sun's galactic travel are not the same thing.

In the last picture of @HanDeBruijn above the 'Interstallar medium' is not in the direction of the sun's path, but points toward the galactic centre. Depending on rotation, the sun's galactic travel is likely 'up' in that picture.
In @mikehelland 's picture, the 'Interstallar medium' would be into the page (or out) depending where the galactic centre is, while the sun's travel is shown to the left.

Now thinking about it, it makes sense. The sun orbits in the galactic plane, yes, but so does everything else. Kepler 101. So there should be no orbit headwind. 
The 'wind' comes from all the other stars, of which the majority are toward the galactic centre, and our heliotail points away from the galactic centre, like a typical comet tail would point away from the sun. 

Apologies if this was obvious to everyone else ...

**Reply by mikehelland** [reply]
Ah!

Well, if that's the case "bow wave" doesn't seem like the best name.

**Reply by mikehelland** [reply]
I'm wondering if anyone can find some references to back up what Francois has said? I find it plausible, but in general, that's not how it's described:

>On a broader scale, the motion of the heliosphere through the fluid medium of the ISM results in an overall comet-like shape. The solar wind plasma which is moving roughly "upstream" (in the same direction as the Sun's motion through the galaxy) is compressed into a nearly-spherical form, whereas the plasma moving "downstream" (opposite the Sun's motion) flows out for a much greater distance before giving way to the ISM, defining the long, trailing shape of the heliotail.

>A proposed model hypothesizes that the heliosheath is shaped like the coma of a comet, and trails several times that distance in the direction opposite to the Sun's path through space.

https://en.wikipedia.org/wiki/Heliosphere

That could just be wikipedia being wrong though.

An interesting point... however, follows that quote:

>Voyager project scientists have determined that the heliosheath is not "smooth" – it is rather a "foamy zone" filled with magnetic bubbles, each about 1 AU wide.

Hmmmm. 1 AU wide... about 100 AU away.... is this the cause of the anisotropies and the peak in the angular power spectrum?

**Comment by mikehelland** [comment]
https://en.wikipedia.org/wiki/Heliosphere

>As part of the interplanetary magnetic field, the heliosphere shields the Solar System from significant amounts of cosmic ionizing radiation

Surely the energy from the cosmic rays toward us, and the solar winds away from us, must go somewhere. Wouldn't that energy be radiated away?
